ansible-inside/stories/assembly-ansible-inside-install-registry.adoc

```diff
@@ -0,0 +1,14 @@
+ifdef::context[:parent-context-of-ansible-inside-install-registry: {context}]
+[id="ansible-inside-install-registry_{context}"]
+
+= Downloading {ExecEnvShort}s from the Red Hat Registry
+
+{ExecEnvNameStart} are used in {AnsibleInsideNameShort} as either a baseline to execute your Ansible automation with,
+or can be used with {Builder} to provide extended automation compatibility to suit your needs.
+
+include::topics/ref-ansible-inside-rhel9-ees.adoc[leveloffset=+1]
```

Comment:
Might be worth leaving a line between these for later.

Reply:
Good thinking - thanks